src/transformers/models/flaubert/modeling_flaubert.py

```diff
@@ -130,12 +296,165 @@
     "The bare Flaubert Model transformer outputting raw hidden-states without any specific head on top.",
     FLAUBERT_START_DOCSTRING,
 )
-class FlaubertModel(XLMModel):
+# Copied from transformers.models.xlm.modeling_xlm.XLMPredLayer with XLM->Flaubert
+class FlaubertPredLayer(nn.module):
```

Comment:
```suggestion
class FlaubertPredLayer(nn.Module):
```
You have a typo here.